src/Select/SelectInput.js

```diff
@@ -343,6 +349,14 @@ class SelectInput extends React.Component<AllProps, State> {
             ...MenuProps.MenuListProps,
             role: 'listbox',
           }}
+          PaperProps={{
+            ...MenuProps.PaperProps,
+            style: {
+              minWidth:
+                this.state.anchorEl != null && !autoWidth ? this.state.anchorEl.clientWidth : 0,
```

Comment:
We might don't want to inline the logic given it start to be complexe.

Reply:
> It would be good to have a test and a demo for this new feature.

@oliviertassinari Indeed, but I don't know how to test the width of the component. Is this even possible without _actually_ rendering the component in a browser?